docs/guide/areas-and-widgets/README.md

```diff
@@ -0,0 +1,147 @@
+# Areas and widgets
+
+**Areas** are core to Apostrophe's in-context editing experience. They are special fields where editors can add one or more content widgets. A **widget** is a section of structured content, such as a block of rich text, an image slideshow, or a grid of featured products. Together, these two features let editors add custom and advanced content to a website, move it around, and edit it &ndash; all within a defined modal that maintains the content design.
+
+## Basic area configuration
+
+Like other fields, you will configure an area as part of the [field schema](/guide/content-schema.md) for a page or piece type. Apostrophe comes with several widgets that you can use right away, including ones for adding rich text, inserting an image, and embedding a video.
```

Comment:
I think we could add a sentence about "areas accept widgets". I never advocate for adding sentence but it may make sense here between the first and second.

Reply:
alternatively I see you say this more or less below, so maybe we could nix the second sentence.

Reply:
Move "Every area needs a `widgets`" up here?